app/templates/components/billing-resubscribe-button.hbs

```diff
@@ -1,8 +1,11 @@
 {{#if this.hasSubscriptionPermissions}}
   {{#if this.canResubscribe}}
-    <button onClick={{action (perform this.resubscribe)}} class='button--green' data-test-resubscribe-subscription='true'>
+    <button onClick={{action (perform this.resubscribe)}} class='button--green mr-1' data-test-resubscribe-subscription='true'>
```

Comment:
Since we're making an update to this, might as well change it to use the `on` modifier for the action, like: `<button {{on 'click' (action (perform this.resubscribe))}}`.